peodd/peodd.py

```diff
@@ -58,7 +62,10 @@ def main(output):
             msg = "{} file is not valid".format(pyproject_file)
             raise click.ClickException(msg)
 
-    poetry_dev_dependencies = toml_dict["tool"]["poetry"]["dev-dependencies"]
+    if main:
+        poetry_dev_dependencies = toml_dict["tool"]["poetry"]["dependencies"]
```

Comment:
I would rename this variable `poetry_dev_dependencies` since it is not dev_dependencies anymore.
```
-         poetry_dev_dependencies = toml_dict["tool"]["poetry"]["dependencies"]
+         dependencies = toml_dict["tool"]["poetry"]["dependencies"]
```

Reply:
```python
dependencies = toml_dict["tool"]["poetry"]["dependencies"]
```